plugins/modules/xbps.py

```diff
@@ -40,13 +40,14 @@
     type: str
   recurse:
     description:
-      - When removing a package, also remove its dependencies, provided that they are not required by other packages and were not explicitly installed
-        by a user.
+      - When removing a package, also remove its dependencies, provided that they are not required by other packages and were
+        not explicitly installed by a user.
```

Comment:
Did you change something in the andebox code, or do the reformattings just happen despite the previous normalization? (I got the feeling that this might be the latter, and reformatting is not idempotent...)

Reply:
That being said, I also made some changes and things like id -> ID are now fixed automatically. I am keeping an eye to make sure.